LayoutTests/fast/webgpu/nocrash/fuzz-282485.html

```diff
@@ -30,52 +29,53 @@
   var out: FragmentOutput0;
   return out;
 }
-`
-  });
-  pipeline0 = await device0.createRenderPipelineAsync({
-    layout : pipelineLayout0,
-    fragment : {
-      module : shaderModule1,
-      entryPoint : 'fragment0',
-      targets : [ {format : 'rgba16float'} ]
-    },
-    vertex : {
-      module : shaderModule1,
-      buffers : [ {
-        arrayStride : 0,
-        attributes : [
-          {format : 'uint16x4', offset : 0, shaderLocation : 0},
-          {format : 'unorm10-10-10-2', offset : 0, shaderLocation : 1}
-        ]
-      } ]
-    }
-  });
-  texture14 = device0.createTexture({
-    size : {width : 1, height : 1, depthOrArrayLayers : 1},
-    dimension : '3d',
-    format : 'rgba16float',
-    usage : GPUTextureUsage.RENDER_ATTACHMENT
-  });
-  textureView16 = texture14.createView();
-  buffer12 = device0.createBuffer({size : 10, usage : GPUBufferUsage.INDEX});
-  commandEncoder29 = device0.createCommandEncoder();
-  renderPassEncoder2 = commandEncoder29.beginRenderPass({
-    colorAttachments : [ {
-      view : textureView16,
-      depthSlice : 0,
-      loadOp : 'clear',
-      storeOp : 'discard'
-    } ],
-  });
-  renderPassEncoder2.setPipeline(pipeline0)
-  renderPassEncoder2.setIndexBuffer(buffer12, 'uint16')
-  renderPassEncoder2.setVertexBuffer(0, buffer4);
-  renderPassEncoder2.drawIndexed(3)
-  renderPassEncoder2.end()
-  commandBuffer3 = commandEncoder29.finish();
-  device0.queue.submit([ commandBuffer3 ])
+  `});
+    pipeline0 = await device0.createRenderPipelineAsync({
+      layout : pipelineLayout0,
+      fragment : {
+        module : shaderModule1,
+        entryPoint : 'fragment0',
+        targets : [ {format : 'rgba16float'} ]
+      },
+      vertex : {
+        module : shaderModule1,
+        buffers : [ {
+          arrayStride : 0,
+          attributes : [
+            {format : 'uint16x4', offset : 0, shaderLocation : 0},
+            {format : 'unorm10-10-10-2', offset : 0, shaderLocation : 1}
+          ]
+        } ]
+      }
+    });
+    texture14 = device0.createTexture({
+      size : {width : 1, height : 1, depthOrArrayLayers : 1},
+      dimension : '3d',
+      format : 'rgba16float',
+      usage : GPUTextureUsage.RENDER_ATTACHMENT
+    });
+    textureView16 = texture14.createView();
+    buffer12 = device0.createBuffer({size : 10, usage : GPUBufferUsage.INDEX});
+    commandEncoder29 = device0.createCommandEncoder();
+    renderPassEncoder2 = commandEncoder29.beginRenderPass({
+      colorAttachments : [ {
+        view : textureView16,
+        depthSlice : 0,
+        loadOp : 'clear',
+        storeOp : 'discard'
+      } ],
+    });
+    renderPassEncoder2.setPipeline(pipeline0)
+    renderPassEncoder2.setIndexBuffer(buffer12, 'uint16')
+    renderPassEncoder2.setVertexBuffer(0, buffer4);
+    renderPassEncoder2.drawIndexed(3)
+    renderPassEncoder2.end()
+    commandBuffer3 = commandEncoder29.finish();
+    device0.queue.submit([ commandBuffer3 ])
```

Comment:
Missing semi-colon at the end of this line `device0.queue.submit([ commandBuffer3 ])`